helpers/filterSeats.js

```diff
@@ -21,8 +24,11 @@ module.exports = function(dust) {
     var venueProduct = _.head(params.venueProducts);
     var ticketRates = params.ticketRates;
     var roomRates = params.roomRates;
-    var bandColours = params.bandColours;
-    var sortByInput = params.sortBy;
+    var sortType = params.sortType;
+    var seatLegend = params.seatLegend;
+
+    var bestPrice = {};
+    var currentPrice, topTicket;
```

Comment:
And what about using some destructering here? 
```
var {ticketRates, roomRates, sortType, seatLegend} = params;
```

Unless this doesn't get transpiled and might not work on older browsers (IE)